src/core/lib/gprpp/thd.h

```diff
@@ -64,9 +64,19 @@ class Thread {
     }
     bool tracked() const { return tracked_; }
 
+    /// Set thread stack size (in bytes). Set to 0 will reset stack size to
+    /// default value, which is 64KB for Windows threads and 2MB for Posix(x86)
```

Comment:
"reset" sounds that you are changing the stack size from its original value.

How about "Set to 0 will use the default stack size"?

Reply:
And also "Sets thread stack size".